msteams-platform/concepts/design/design-teams-app-light-box-view.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Design your app with lightbox UI components
+author: v-npaladugu
+description: Learn how to use lightbox UI component from Teams UI kit to build Microsoft Teams apps.
+ms.localizationpriority: medium
+ms.topic: reference
+ms.author: surbhigupta
+ms.date: 01/29/2023
+---
+
+# Designing lightbox for your Microsoft Teams app
+
+The lightbox is a display component that emphasizes important information by deactivating the page layout behind it. Information in the lightbox is non-editable and non-interactive. The lightbox view enables users to preview media content, including images, videos, audio files, documents, and PDFs, on a large surface over the main Teams window. 
+
+The lightbox component is designed for quick reviews, understanding, or verification. 
+It don't support actions for editing or collaboration. 
+The most frequently used file types in the lightbox are images, short PDFs or documents, and short video clips. 
+
+Users can initiate the lightbox in a new Teams child window or open it directly in a native app or browser. 
+
+# [Desktop](#tab/desktop)
+
+:::image type="content" source="../../assets/images/light-box/lightbox-desktop.png" alt-text="Example shows the default lightbox view in the Teams desktop.":::
+
+# [Mobile](#tab/mobile)
+
+:::image type="content" source="../../assets/images/light-box/lightbox-mobile.png" alt-text="Example shows the default lightbox view in the Teams mobile.":::
+
+---
+
+## Anatomy
+
+Lightbox view includes a header, center stage, and carousel.
+
+:::image type="content" source="../../assets/images/light-box/lightbox-anatomy.png" alt-text="Design anatomy of the lightbox view in the Teams." lightbox="../../assets/images/light-box/lightbox-anatomy.png":::
+
+|Counter|Description|
+|----------|-----------|
+|1|**Back**: Button to navigate to the previoius screen|
+|2|**Open in**: Button (basic or split) to navigate to the external window|
+|3|Buttons with Content actions: Zoom in or out, Share, Download, Popout (open new child window), open chat panel|
+|4|Chevrons for slide show|
+|A|**Lightbox Header**: Actionable|
+|B|**Center stage**: Main content container|
+|C|**Carousel**: Actionable|
+
+## Sizing of the components
+
+Default sizing for the container with main content. Following is the default content:
+
+# [Image](#tab/image)
+
+:::image type="content" source="../../assets/images/light-box/image-sizing.png" alt-text="Example shows an image sizing with lightbox view in the Teams." lightbox="../../assets/images/light-box/image-sizing.png":::
+
+# [Carousel](#tab/carousel)
+
+Contains all media files from the current chat, channel post, or file folder. 
+
+:::image type="content" source="../../assets/images/light-box/carousel-sizing.png" alt-text="Example shows carousel sizing with lightbox view in the Teams." lightbox="../../assets/images/light-box/carousel-sizing.png":::
```

Comment:
Check the image, the arrow is overlapping, and the carousel is nor centered. You will need to update the image and explain the same.